src/components/Dashboard.js

```diff
@@ -0,0 +1,254 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+const Dashboard = () => {
+  const [service, setService] = useState('');
+  const [externalId, setExternalId] = useState(uuidv4());
+  const [serviceName, setServiceName] = useState('');
+  const [roleArn, setRoleArn] = useState('');
+  const [externalIdInput, setExternalIdInput] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleServiceChange = (e) => {
+    setService(e.target.value);
+  };
+
+  const handleServiceNameChange = (e) => {
+    setServiceName(e.target.value);
+  };
+
+  const handleRoleArnChange = (e) => {
+    setRoleArn(e.target.value);
+  };
+
+  const handleExternalIdInputChange = (e) => {
+    setExternalIdInput(e.target.value);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const data = { service_name: serviceName, role_arn: roleArn, external_id: externalIdInput };
+    
+    try {
+      const response = await fetch('http://localhost:5000/create_role', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+      });
+      
+      if (response.ok) {
+        const result = await response.json();
+        setMessage('Role created successfully');
+        console.log('Role created:', result);
+      } else {
+        setMessage('Error creating role');
+      }
+    } catch (error) {
+      setMessage('Error creating role');
+      console.error('Error:', error);
+    }
+  };
+
+  const redirectToAWSRoleCreation = () => {
+    const awsAccountNumber = '767398108291';
+    setExternalId(uuidv4());
```

Comment:
yah for now , just for demo u can use this account , after we crete aws organisations we can extend this